CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## 1.2.0 / 2016-10-07
+
+* [FEATURE] Enable the use of query parameters in `/graph` URLs.
+* [FEATURE] PromQL: Add `minute()` function.
+* [FEATURE] Add GCE service discovery.
+* [FEATURE] Allow job names to start with a digit.
+* [FEATURE] Allow disabling local storage.
+* [FEATURE] EC2 service discovery: Expose `ec2_instance_state`.
+* [FEATURE] HTTP basic auth and TLS support for remote write path.
+* [FEATURE] Support for relabelling in remote write path.
+* [ENHANCEMENT] Add `go_import_path` to Travis so it works on a fork.
```

Comment:
If you believe it's important to mention such commits, I'd use a different group for such changes and move it to the bottom.